step.yml

```diff
@@ -1,8 +1,16 @@
-title: Generic File Storage
-summary: The step will download all your files you uploaded in the GENERIC FILE STORAGE section of the Code Signing tab in your Workflow Editor.
-description: |
-  The step will download all your files you uploaded in the GENERIC FILE STORAGE section of the Code Signing tab in your Workflow Editor with the same name, under `$GENERIC_FILE_STORAGE` path.
-  As an example, if you upload `myconfig.ini` to GENERIC FILE STORAGE, then the step will download it and it will be available as `$GENERIC_FILE_STORAGE/myconfig.ini`
+title: "Generic File Storage"
+summary: Download all files you uploaded in the **GENERIC FILE STORAGE** section of the Code Signing tab in your Workflow Editor.
```

Comment:
I think this title is too long (it will be visible in the step browser in the workflow editor). I'd suggest something shorter like:
```
Download all files uploaded to Generic File Storage.
```
Wdyt?

Reply:
It's okay to me!